datastore/datastore.go

```diff
@@ -0,0 +1,322 @@
+/*
+the datastore package retrieves data from the stackdriver API as well as query of '__Stat_Kind__' from datastore directly.
+datastore.go contains the logic to send data to new relic and the datastore query metric/connection. stackdrive.go contains
+the stackdriver API metrics/connection.
+*/
+
+package datastore
+
+import (
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"cloud.google.com/go/datastore"
+	"github.com/GannettDigital/go-newrelic-plugin/helpers"
+	"github.com/Sirupsen/logrus"
+	"github.com/buger/jsonparser"
+	"golang.org/x/net/context"
+	"golang.org/x/oauth2"
+	"golang.org/x/oauth2/google"
+	"google.golang.org/api/monitoring/v3"
+	"google.golang.org/api/option"
+)
+
+const (
+	// NAME - name of plugin
+	NAME string = "datastore"
+
+	// ProtocolVersion -
```

Comment:
minor but I would say that is a useless comment and can be removed. Also in case you didn't know for these constants you don't need to specify they are a string the quotes imply that, either way works though.